app/src/main/java/com/babylon/wallet/android/data/repository/cache/database/ProvidedConverters.kt

```diff
@@ -37,23 +109,23 @@ class StateDatabaseConverters {
     // Behaviours
     @TypeConverter
     fun stringToBehaviours(string: String?): BehavioursColumn? {
-        return string?.let { BehavioursColumn(behaviours = json.decodeFromString(string)) }
+        return string?.let { json.decodeFromString(string) }
```

Comment:
maybe we should use same `json` as we use for gateway models serialization?

Reply:
No it is different, this is how the column is serialized into a json string. Room stores metadata in a custom type called `MetadataColumn` which uses this serialisation logic inside. Also the `Metadata` type we have defined is a domain model and not a GW one.